.github/workflows/release.yml

```
@@ -0,0 +1,35 @@
+
+    - name: Build binaries
+      run: |
+        mkdir -p dist
+        GOOS=linux GOARCH=amd64 go build -ldflags="-s -w" -o dist/audnexus-provider-linux-amd64 ./cmd/server
+        GOOS=linux GOARCH=arm64 go build -ldflags="-s -w" -o dist/audnexus-provider-linux-arm64 ./cmd/server
+        GOOS=darwin GOARCH=amd64 go build -ldflags="-s -w" -o dist/audnexus-provider-darwin-amd64 ./cmd/server
+        GOOS=darwin GOARCH=arm64 go build -ldflags="-s -w" -o dist/audnexus-provider-darwin-arm64 ./cmd/server
+        GOOS=windows GOARCH=amd64 go build -ldflags="-s -w" -o dist/audnexus-provider-windows-amd64.exe ./cmd/server
+
+    - name: Create Release
+      uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)